SetupDataPkg/ConfApp/ConfApp.h

```diff
@@ -62,6 +62,7 @@ typedef enum SetupConfState_t_def {
   SetupConfUpdateSerial,
   SetupConfDumpSerial,
   SetupConfDumpComplete,
+  SetupConfError,

```

Comment:
nit: Is there anything preventing moving the new SetupConfError to the end of the enum?

Reply:
You mean above the SetupConfMax? No, I just wanted to make the enum definition order and the switch case order to be in sync. I can move this downwards easily.

Reply:
sorry, I meant that SetupConfError is new, and it was inserted Infront of SetupConfExit instead of after SetupConfgExit.

I through SetupConfMax should stay at the end of the enum.

Its minor because the usage of the enum is only internally to this module.

Reply:
Sure, I understand your point. It would not make any substantial difference. But will update the order in the next update.

Reply:
Updated.